bin/varnishd/cache/cache_req.c

```diff
@@ -72,6 +73,37 @@ Req_AcctLogCharge(struct VSC_main_wrk *ds, struct req *req)
 	memset(a, 0, sizeof *a);
 }
 
+void
+Req_LogHit(struct worker *wrk, struct req *req)
+{
+	struct objcore *oc;
+	struct boc *boc;
+	const char *clen, *sep;
+
+	CHECK_OBJ_NOTNULL(wrk, WORKER_MAGIC);
+	CHECK_OBJ_NOTNULL(req, REQ_MAGIC);
+	oc = req->objcore;
+	CHECK_OBJ_NOTNULL(oc, OBJCORE_MAGIC);
+	boc = HSH_RefBoc(req->objcore);
```

Comment:
um. Maybe I did not understand the impact of your earlier question on the whether to log `len_so_far`, but taking another boc ref just for logging sounds like something we should avoid.

Reply:
We could avoid the boc reference but would it be safe to read `len_so_far`? Getting the content length from the objcore should remain safe regardless.

Reply:
I think we should really avoid it. If this feature required another bocref, I'd rather not have it.

Reply:
> but would it be safe to read `len_so_far`?